pkg/plugin/v3/scaffolds/internal/templates/config/kdefault/manager_auth_proxy_patch.go

```diff
@@ -63,8 +66,10 @@ spec:
         ports:
         - containerPort: 8443
           name: https
-      - name: manager
+      {{- if not .ComponentConfig }}
+			- name: manager
         args:
         - "--metrics-addr=127.0.0.1:8080"
         - "--enable-leader-election"
+			{{- end }}
```

Comment:
I would either do this:
```suggestion
      {{ if not .ComponentConfig -}}
      - name: manager
        args:
        - "--metrics-addr=127.0.0.1:8080"
        - "--enable-leader-election"
      {{- end }}
```
or:
```suggestion
{{- if not .ComponentConfig }}
      - name: manager
        args:
        - "--metrics-addr=127.0.0.1:8080"
        - "--enable-leader-election"
{{- end }}
```